indra/llcharacter/llik.h

```diff
@@ -61,29 +61,33 @@ class Solver;
 class Joint;
 
 // config flags
-constexpr U8 CONFIG_FLAG_LOCAL_POS = 1 << 0;
-constexpr U8 CONFIG_FLAG_LOCAL_ROT = 1 << 1;
-constexpr U8 CONFIG_FLAG_LOCAL_SCALE = 1 << 2;
-constexpr U8 CONFIG_FLAG_DISABLE_CONSTRAINT = 1 << 3;
-constexpr U8 CONFIG_FLAG_TARGET_POS = 1 << 4;
-constexpr U8 CONFIG_FLAG_TARGET_ROT = 1 << 5;
-constexpr U8 CONFIG_FLAG_HAS_DELEGATED      = 1 << 6;  // EXPERIMENTAL
-constexpr U8 CONFIG_FLAG_ENABLE_REPORTING  = 1 << 7;  // EXPERIMENTAL
-
-constexpr U8 IK_FLAG_LOCAL_ROT = 1 << 1; // IK has adjusted local_rot
-constexpr U8 IK_FLAG_ACTIVE = 1 << 5;
-constexpr U8 IK_FLAG_LOCAL_ROT_LOCKED = 1 << 7; // local_rot is locked during IK
-
-constexpr U8 MASK_POS = CONFIG_FLAG_TARGET_POS | CONFIG_FLAG_LOCAL_POS;
-constexpr U8 MASK_ROT = CONFIG_FLAG_TARGET_ROT | CONFIG_FLAG_LOCAL_ROT;
-constexpr U8 MASK_TRANSFORM = MASK_POS | MASK_ROT;
-constexpr U8 MASK_LOCAL = CONFIG_FLAG_LOCAL_POS | CONFIG_FLAG_LOCAL_ROT | CONFIG_FLAG_DISABLE_CONSTRAINT;
-constexpr U8 MASK_TARGET = CONFIG_FLAG_TARGET_POS | CONFIG_FLAG_TARGET_ROT;
+using Flag = U16;
+
+constexpr Flag CONFIG_FLAG_LOCAL_POS = 1 << 0;
+constexpr Flag CONFIG_FLAG_LOCAL_ROT = 1 << 1;
+constexpr Flag CONFIG_FLAG_LOCAL_SCALE = 1 << 2;
+constexpr Flag CONFIG_FLAG_DISABLE_CONSTRAINT = 1 << 3;
+constexpr Flag CONFIG_FLAG_TARGET_POS = 1 << 4;
+constexpr Flag CONFIG_FLAG_TARGET_ROT = 1 << 5;
+constexpr Flag CONFIG_FLAG_HAS_DELEGATED      = 1 << 6;  // EXPERIMENTAL
+constexpr Flag CONFIG_FLAG_ENABLE_REPORTING  = 1 << 7;  // EXPERIMENTAL
+constexpr Flag CONFIG_FLAG_IS_ELBOW = 1 << 8;
+
+constexpr Flag IK_FLAG_LOCAL_ROT = 1 << 1; // IK has adjusted local_rot
+constexpr Flag IK_FLAG_ACTIVE = 1 << 5;
+constexpr Flag IK_FLAG_LOCAL_ROT_LOCKED = 1 << 7; // local_rot is locked during IK
+
+constexpr Flag MASK_POS = CONFIG_FLAG_TARGET_POS | CONFIG_FLAG_LOCAL_POS;
+constexpr Flag MASK_ROT = CONFIG_FLAG_TARGET_ROT | CONFIG_FLAG_LOCAL_ROT;
+constexpr Flag MASK_TRANSFORM = MASK_POS | MASK_ROT;
+constexpr Flag MASK_LOCAL = CONFIG_FLAG_LOCAL_POS | CONFIG_FLAG_LOCAL_ROT | CONFIG_FLAG_DISABLE_CONSTRAINT;
+constexpr Flag MASK_TARGET = CONFIG_FLAG_TARGET_POS | CONFIG_FLAG_TARGET_ROT;
```

Comment:
I ran out of bits in the `U8` flags, so I upped it to `U16` and employed a `using` declaration to simplify future changes around that.